tox.ini

```diff
@@ -6,7 +6,7 @@ deps =
     -rrequirements.txt
 setenv =
  PYTHONPATH=.
-commands = {envpython} {envbindir}/run-ci --ignore=tests -c unittests/pytest.ini {posargs:unittests}
+commands = py.test --ignore=tests -c unittests/pytest.ini {posargs:mita oc ocs resources unittests utility ocsci}
```

Comment:
This was intentionally run-ci (due to a comment in old pr) to test the entry-point.

Reply:
That makes sense - however, if we move the logic from the entry point into one or more of the packages themselves, we can simply unit test that logic.

Reply:
Mind dropping this one until we have something more in unittests? :)

Reply:
Sure - I just repushed.

I'm ready to write more unit tests, but there's actually an issue that needs addressing first. Because pytest runs all tests in the same process, modifications to `ocsci.config` will persist between tests. So we need a way to reset the configuration. The current "module as a container for configuration" approach makes that really tough.

I have a decent implementation ready that could fix this using a singleton, but I don't want to tack it onto this PR. We should discuss that soon!